src/hooks/usePublicMessages.js

```diff
@@ -113,3 +125,23 @@ function comparePublicMessages(publicMessage1, publicMessage2) {
     return publicMessage1.message.localeCompare(publicMessage2.message);
   }
 }
+
+/** Compares two routes for sorting purposes.
+ *
+ * @param {Object} route1
+ * @param {Object} route2
+ * @return {Number}
+ * @see {fetchPublicMessages}
+ */
+function compareRoutes(route1, route2) {
+  const [order1, order2] = [route1.sortOrder, route2.sortOrder];
+  if (order1 === null && order2 !== null) {
+    return -1;
+  } else if (order1 !== null && order2 === null) {
+    return 1;
+  } else if (order1 !== null && order2 !== null && order1 !== order2) {
+    return order1 - order2;
```

Comment:
I wouldn't expect `SortOrder` to come in as null when looking through the API - have you encountered that case before?